cupy/core/_routines_sorting.pyx

```diff
@@ -282,7 +278,7 @@ def _partition_kernel(dtype):
             CArray<T, 1, true> a, ptrdiff_t x, ptrdiff_t y, int i) {
         for (ptrdiff_t s = 2; s <= y - x; s *= 2) {
             for (ptrdiff_t w = s / 2; w >= 1; w /= 2) {
-                bitonic_sort_step<T>(a, x, y, i, s, w);
+                bitonic_sort_step< T >(a, x, y, i, s, w);
```

Comment:
Any reason behind this changes?😇

Reply:
This is to specialize for `bitonic_sort_step< complex<float> >` and friend. Without the spacing  the compilation would fail (in a way that the test suite couldn't report).